app/controllers/registration_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RegistrationController < ApplicationController
```

Comment:
This is actually a new file that just has a similar name, but I will keep this in mind for the future.